### ISSUE TITLE
Style: migrate from relative import to path aliasing 

### ISSUE THREAD

**Issue by KemingHe** [wei/socialify#408]
# Code Style Migration Proposal

Propose to migrate all imports to using path aliasing (i.e. `@pages`, `@components`) instead of relative imports (i.e. `../../module.ts`).

## Rationale

* Ease of maintenance. Modularized pathing. Documentation-as-code.
* Setting stage for greater migrations, i.e. NextJS's `Page Router` -> `App Router`.

## Demonstration

> Ref `src/components/configuration/config.tsx`

### Before

```Typescript
import { RepoQueryResponse } from '../../../common/github/repoQuery'
import ConfigContext from '../../contexts/ConfigContext'
```

### After 

```Typescript
import { RepoQueryResponse } from '@common/github/repoQuery'
import ConfigContext from '@contexts/ConfigContext'
```

## Estimated Delivery Time

Less than 1 week by @KemingHe

**Comment by KemingHe** [comment]
This applies to source code as well as `Jest` unit tests, both of which I know exactly the best practice to migrate.

**Comment by KemingHe** [comment]
Resolved in #421 closing issue...
